charts/templates/statefulset.yaml

```diff
@@ -66,6 +66,8 @@ spec:
         env: 
         - name: MARKLOGIC_GROUP
```

Comment:
Same. This should be removed and put it in ConfigMap.

Reply:
MARKLOGIC_GROUP is an env var from Docker, but we also use it to update the default group which docker does not handle. so we might want to use the values file and env mapping?